Makefile

```
@@ -0,0 +1,57 @@
+IMAGE_REGISTRY?=quay.io
+IMAGE_REPOSITORY?=app-sre
+IMAGE_NAME?=backplane-api
+VERSION=$(shell git rev-parse --short=7 HEAD)
+UNAME_S := $(shell uname -s)
+
+IMAGE_URI_VERSION:=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME):$(VERSION)
+IMAGE_URI_LATEST:=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME):latest
+
+GOLANGCI_LINT_VERSION=v1.50.1
+
+CONTAINER_ENGINE:=$(shell command -v podman 2>/dev/null || command -v docker 2>/dev/null)
+RUN_IN_CONTAINER_CMD:=$(CONTAINER_ENGINE) run --platform linux/amd64 --rm -v $(shell pwd):/app -w=/app backplane-api-builder /bin/bash -c
+
+.PHONY: clean
+clean:
+	rm -f backplane-api
+
+.PHONY: lint-in-container
+lint-in-container: build-image
+	$(RUN_IN_CONTAINER_CMD) "go mod download && make lint"
+
+getdeps:
+	@command -v golangci-lint 1>/dev/null || (echo "Installing golangci-lint" && curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell go env GOPATH)/bin $(GOLANGCI_LINT_VERSION))
+
+.PHONY: lint
+lint: getdeps
+	golangci-lint run
+
+test-in-container: build-image
+	$(RUN_IN_CONTAINER_CMD) "make test"
+
+test:
+	go test -v ./...
+	@echo Run \'make test-cover\' to see the test coverage report
+
+test-cover:
+	go test -cover -coverprofile=coverage.out ./...
+	go tool cover -html=coverage.out
+
+generate:
+	go generate -v ./...
+
```

Please add this target as well.
```suggestion
dev-certs:
	bash -c 'openssl req \
		-x509 \
		-out localhost.crt \
		-keyout localhost.key \
		-newkey rsa:4096 \
		-nodes \
		-sha256 \
		-subj "/CN=localhost" \
		-extensions san \
		-config <( \
			echo "[req]"; \
			echo "distinguished_name=req"; \
			echo "[san]"; \
			echo "subjectAltName=DNS:localhost")'
ifeq ($(UNAME_S),Darwin)
	security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain localhost.crt
endif
ifeq ($(UNAME_S),Linux)
	cp localhost.crt /etc/pki/tls/certs/
	cp localhost.crt /etc/pki/ca-trust/source/anchors/
	update-ca-trust extract
endif

```